tools/monitor/live.py

```diff
@@ -0,0 +1,84 @@
+
+import sys
+import os
+from collections import OrderedDict
+
+sys.path.append(os.path.join(os.path.dirname(__file__),
```

Comment:
Jeżeli coś takiego moze się gdzieś znaleźć to tylko w pliku, który jest mainem

Reply:
Usunięte.